packages/components/src/Combobox/Combobox.tsx

```diff
@@ -10,6 +10,14 @@ import {
 
 export interface ComboboxProps {
   readonly children: ReactElement | ReactElement[];
+
+  /**
+   * multi select
+   * @default false
+   * @type boolean
+   * @description if true, allows multiple selections
+   */
+  readonly multiSelect?: boolean;
```

Comment:
Is there a reason why we explicitly set `@description` and `@type`? We should be able to follow our normal props pattern:

```suggestion
  /**
   * When `true`, `Combobox` will allow for multiple selections
   *
   * @default false
   */
  readonly multiSelect?: boolean;
```

Reply:
ah ok!!!! 👌🏻